### PR TITLE
Progress Bar

### LINKED ISSUE #1519: <ProgressBar />
```js
<ProgressBar
  title={string}
  total={string}
  percentDone={?number}
  theme={?string = "light" || "dark" } // default dark
  style={?object}
/>
```

Supporting Image:
screen shot 2016-12-01 at 2 55 16 pm

### PR DESCRIPTION
**Pull request by JakeDawkins**
Resolves #1519. This implementation takes the `total` prop exactly as passed, and does no trimming of decimal places or validation. There's an open discussion of this in #1519 

screen shot 2016-12-05 at 6 23 46 pm
screen shot 2016-12-05 at 6 24 02 pm

### CODE REVIEW

**Comment by jbaxleyiii** [comment]
**[Storybook Hub](https://hub.getstorybook.io/)** is building storybooks for every commit in this PR.

**Latest Storybook: https://588b129f-c4d4-49b0-b8d9-1665c887652d.sbook.io/**
<sub>( Visit [here](https://hub.getstorybook.io/app/github.com/NewSpring/Holtzman/branch/progress-bar/) for more info )</sub>

> _This comment is automatically added by Storybook Hub_

**Comment by jbaxleyiii** [comment]
@JakeDawkins letting the parent decided layout of this sounds good to me! I could see this component in other use cases where things like % or decimals may be wanted. Better to let the parent decide

File: imports/components/progressBar/__stories__/progress-bar.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# ProgressBar
+
+## Usage
+
+```
+<ProgressBar
+  title="My Fund"
+  total="100"
+  percentDone= 50
```
Comment (jbaxleyiii) [inline imports/components/progressBar/__stories__/progress-bar.md]: This is minor, but this is has extra spacing

**Comment by JakeDawkins** [comment]
Is that what you were talking about @jbaxleyiii

**Comment by jbaxleyiii** [comment]
@JakeDawkins 💯

**Pull request merged**